packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -255,6 +255,9 @@ function getDefaultHandlers(modules: ValidatorFnsModules, options: GossipHandler
         chain.seenGossipBlockInput.prune();
       })
       .catch((e) => {
+        // Any unexpected error
+        let logLevel = LogLevel.error;
```

Comment:
do not initialize with anything, this way typescript will tell you if any case or switch has not been covered and is helpful in things being a bit more declarative, rest looks good :+1:

Reply:
Good point, this ensure we have to explicitly handle each case